docs/resources/how-it-works.md

```diff
@@ -67,7 +67,7 @@ Currently, the Lit network is in [Mainnet Beta](../network/migration-guide.md) a
 
 ### Node Operators
 
-The current node operators active on the Habanero Mainnet Beta include:
+The current node operators active on the Datil Mainnet Beta include:
```

Comment:
@a1ttech to confirm this node operator list below for Datil

Reply:
Changed node operator list